VMware.CloudFoundation.PowerManagement.psm1

```diff
@@ -744,6 +744,104 @@ Function Set-VsanClusterPowerStatus {
 }
 Export-ModuleMember -Function Set-VsanClusterPowerStatus
 
+Function Invoke-VxrailClusterShutdown{
```

Comment:
```suggestion
Function Invoke-VxrailClusterShutdown {
```

Reply:
addressed in commit f8c5a69ef84a1a40675a91879ad0a67701fba108